documentation/source/newsfragments/3191.bugfix.rst

```diff
@@ -0,0 +1 @@
+Updated test_sv_interface with a workaround for Verilator so the test now passes with that simulator
```

Comment:
I'm not sure this is newsworthy.

Reply:
makes sense. The comments in the Pull request dialog talking about steps necessary for a pull request to be accepted made it seem like newsfragments were mandatory.

Reply:
They aren't. That being said, thank you for the PR